PublishTestResultScreenshotV0/index.ts

```diff
@@ -26,9 +27,16 @@ async function run() {
         let authHandler = azdev.getPersonalAccessTokenHandler(authToken);
         let connection = new azdev.WebApi("https://dev.azure.com/" + getOrganization(), authHandler);
         testApi = await connection.getTestApi();
-        await testApi.getTestResultsByBuild(project, +tl.getVariable("Build.BuildId"), undefined, [TestOutcome.Failed])
-            .then(async failedTests =>  uploadScreenshots(failedTests))
-            .catch(err => tl.setResult(tl.TaskResult.Failed, err.message))
+
+        let system = tl.getVariable("SYSTEM");
```

Comment:
In order to make it runnable outside of Azure DevOps, it would be best to add the newly referenced variables with all the others on top of the file
```
// process.env.RELEASE_RELEASEID = ""
// process.env.RELEASE_ENVIRONMENTID = ""
// process.env.SYSTEM = "" // values are build or release
```

Reply:
Yes, you are right.